ui/app/pages/confirm-approve/confirm-approve-content/index.scss

```diff
@@ -292,6 +292,35 @@
       margin-left: 16px;
     }
   }
+
+  &__custom-nonce {
+    width: 100%;
+
+    &__content {
+      margin-bottom: 2px;
+      margin-top: 8px;
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      align-items: center;
+    }
+
+    &__label {
+      @include H7;
+
+      font-weight: 500;
+      color: $scorpion;
```

Comment:
$scorpion, #5d5d5d, is not a part of our color system. If the design calls for this color let's only extend it's usage if it is purposeful to match existing styles within this page. I'm sure that's the case but just want to confirm.

Reply:
This changes will be replaced soon, I am still working on the new design proposed. I had to do a rebase in between because my workspace started behaving weirdly. 
I apologies for taking up your time.

Reply:
@NiranjanaBinoy no worries at all! ping me when you're ready for another look :)